packages/next/src/experimental/testmode/playwright/README.md

```diff
@@ -0,0 +1,96 @@
+# Experimental test mode for Playwright
+
+### Prerequisites
+
+You have a Next.js project.
+
+### Install `@playwright/test` in your project
+
+```sh
+npm install -D @playwright/test
+```
+
+### Optionally install MSW in your project
+
+[MSW](https://mswjs.io/) can be helpful for fetch mocking.
+
+```sh
+npm install -D msw
+```
+
+### Update `playwright.config.ts`
+
+```javascript
+import { defineConfig } from 'next/experimental/testmode/playwright'
+
+export default defineConfig({
+  webServer: {
+    command: 'pnpm dev -- --experimental-test-proxy',
```

Comment:
nit: using `npm install` and `pnpm` here -- maybe we just stick with `npm` for consistency?